### ISSUE TITLE
Hello,I have a question want to ask you !

### ISSUE THREAD

**Issue by BCWang93** [andy-yun/pytorch-0.4-yolov3#61]
Hi,I want to know where is the yolov3_loss file,and which is the yolov2_loss file.I have some doubt about the loss function calculate about yolov3.Thanks!

**Comment by andy-yun** [comment]
@BCWang93 you should check region_layer.py and yolo_layer.py.  Losses are aggregated at forward() function.
